[2022-01-30 01:33:11,156] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag_fhv_2.download_dataset_task scheduled__2020-03-02T06:00:00+00:00 [queued]>
[2022-01-30 01:33:12,574] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag_fhv_2.download_dataset_task scheduled__2020-03-02T06:00:00+00:00 [queued]>
[2022-01-30 01:33:12,574] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-30 01:33:12,575] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-01-30 01:33:12,575] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-30 01:33:15,956] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2020-03-02 06:00:00+00:00
[2022-01-30 01:33:15,989] {standard_task_runner.py:52} INFO - Started process 838 to run task
[2022-01-30 01:33:16,048] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag_fhv_2', 'download_dataset_task', 'scheduled__2020-03-02T06:00:00+00:00', '--job-id', '76', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag_2_fhv.py', '--cfg-path', '/tmp/tmpw5vxsfgf', '--error-file', '/tmp/tmpee9b9b80']
[2022-01-30 01:33:16,049] {standard_task_runner.py:77} INFO - Job 76: Subtask download_dataset_task
[2022-01-30 01:33:21,085] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag_fhv_2.download_dataset_task scheduled__2020-03-02T06:00:00+00:00 [running]> on host 04ca2676bdcc
[2022-01-30 01:33:24,709] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-01-30 01:33:28,706] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag_fhv_2
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2020-03-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-03-02T06:00:00+00:00
[2022-01-30 01:33:29,309] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-30 01:33:30,291] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sSLf https://nyc-tlc.s3.amazonaws.com/trip+data/fhv_tripdata_2020-03.csv > /opt/***/fhv_tripdata_2020-03.csv']
[2022-01-30 01:33:30,305] {subprocess.py:85} INFO - Output:
[2022-01-30 01:33:36,491] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-30 01:33:38,933] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag_fhv_2, task_id=download_dataset_task, execution_date=20200302T060000, start_date=20220130T013312, end_date=20220130T013338
[2022-01-30 01:33:44,901] {local_task_job.py:212} WARNING - State of this instance has been externally set to success. Terminating instance.
